.github/workflows/ci.yml

```diff
@@ -19,3 +21,8 @@ jobs:
     uses: smallstep/workflows/.github/workflows/goCI.yml@main
     with:
       os-dependencies: "libpcsclite-dev"
```

Comment:
We don't do CGO-enabled builds for step-ca, so this dependency is unnecessary.

Reply:
I tried taking it out and the CI workflows failed in multiple places where the package was expected.

Reply:
This is because CGO is enabled here, but installing this dependency is okay.